src/common/k8s-api/api-kube.ts

```diff
@@ -11,4 +11,7 @@ export const apiKubeInjectionToken = getInjectionToken<KubeJsonApi>({
   id: "api-kube-injection-token",
 });
 
+/**
+ * @deprecated Switch to using apiKubeInjectionToken instead
+ */
 export const apiKube = asLegacyGlobalForExtensionApi(apiKubeInjectionToken);
```

Comment:
Note: Looks like this isn't directly exposed in extension API, and it shouldn't therefore be a legacy global at all.